instrumentation/aws-sdk/aws-sdk-1.11/javaagent-unit-tests/src/test/java/TracingRequestHandlerTest.java

```diff
@@ -61,5 +67,8 @@ void shouldSetScopeIfClientSpanNotPresent() {
     // then - no exception and scope not set
     assertThat(request.getHandlerContext(TracingRequestHandler.SCOPE)).isNotNull();
     underTest.afterResponse(request, response(request));
```

Comment:
Hmm, why doesn't `afterResponse` close the scope on its own? Is it really okay that it's only closed on error? CC @anuraaga